### ISSUE TITLE
Type-8 Thermals

### ISSUE THREAD

**Issue by ultimatespirit** [taleden/EDSY#31]
I noticed that the T8's thermal data is still not (publicly) known / included in the EDSY data. As I was unable to find a summary of how your thermal testing procedure works to be able to just give you data, wanted to ask if there was anything we could do to help get T8 thermal data researched. Especially if you'd be willing to disclose how to go about deriving that data independently when required, since my understanding is the original ship thermal post on the ED forums from 2016 is subtly incorrect with its heat model, due to ships having per-ship heat dissipation rates separate of their heat capacity which that thread mainly worked to derive. I wasn't able to find documentation on how best to go about deriving those rates though, and I also saw some posts from a few years back that seemed to imply some of the data was coming from FDev being more open to just releasing hard stats, so unsure if that's just how we're getting the data and fdev just haven't done that for the T8 yet or something.

Tl;dr: I have a Type-8 and am willing to do (and want to do) in game tests to confirm thermal parameters of the ship if it's needed.

**Comment by taleden** [comment]
Estimated thermal properties are now in for the Type-8 (and Mandalay). Thanks!
